mmpose/models/backbones/cpm.py

```diff
@@ -75,6 +77,7 @@ def __init__(self,
                  middle_channels=32,
                  num_stages=6,
                  norm_cfg=dict(type='BN', requires_grad=True)):
+        norm_cfg = copy.deepcopy(norm_cfg)
```

Comment:
need to add that line of comment, so that future reader know what this line is for

Reply:
Apply to all?

Reply:
yea

Reply:
like this

![image](https://user-images.githubusercontent.com/9464825/98895050-1b592280-24e1-11eb-93b4-f56c7922bf22.png)

Reply:
Yes I see.